SimPEG/VRM/ProblemVRM.py

```diff
@@ -0,0 +1,686 @@
+from SimPEG import Problem, mkvc, Maps, Props, Survey
+from SimPEG.VRM.SurveyVRM import SurveyVRM
+import numpy as np
+import scipy.sparse as sp
+
+############################################
+# BASE VRM PROBLEM CLASS
+############################################
+
+
+class BaseProblemVRM(Problem.BaseProblem):
+    """
```

Comment:
can you please indent this over so that the indentation level of the text matches the quotes and use the sphinx formatting for defining your variables (see for example https://github.com/simpeg/simpeg/blob/master/SimPEG/EM/FDEM/ProblemFDEM.py#L63) , eg
```
    """
    Base class for VRM problem

    **Required Arguments**:
    :param discretize.BaseMesh mesh: a 3D tensor or OcTree mesh

    **Optional Inputs**:
    :param int refFact: maximum refinement factor for sensitivities 
    """
```

Reply:
`refFact` --> `ref_fact`: https://www.python.org/dev/peps/pep-0008/